meta-ostro/conf/distro/include/ostro-os-development.inc

```diff
@@ -15,3 +15,16 @@ OSTRO_EXTRA_MOTD () {
 # Everything ready, avoid initial sanity check.
 OSTRO_IMAGE_BUILD_MODE_SELECTED = "1"
 
+# Typical cases for normal users are to produce uncompressed images
+# either for the real target or for the virtual machine. Or both.
+# Note: the variable below uses a hack, which causes the parameter to
+# contain the string "VM" even when it refers to real machines.
+
+# Image for UEFI devices.
+# OSTRO_VM_IMAGE_TYPES = " dsk"
+
+# Image for VirtualBox.
+OSTRO_VM_IMAGE_TYPES = " dsk.vdi"
+
+# Both images.
+# OSTRO_VM_IMAGE_TYPES = " dsk dsk.vdi"
```

Comment:
On 15 March 2016 at 21:06, Patrick Ohly <notifications@github.com> wrote:
>
> The purpose of ostro-os-development.inc and ostro-os-production.inc was to
> switch between production and developer images,
>

ok, ​I had a different interpretation, but I can change the patches to this
one.



-- 
igor